components/resources/library/src/iosMain/kotlin/org/jetbrains/compose/resources/ResourceEnvironment.ios.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.compose.resources
+
+import platform.Foundation.*
+import platform.UIKit.UIScreen
+import platform.UIKit.UIUserInterfaceStyle
+
+internal actual fun getResourceEnvironment(): ResourceEnvironment {
+    val locale = NSLocale.currentLocale()
+
+    val mainScreen = UIScreen.mainScreen
+    val isDarkTheme = mainScreen.traitCollection().userInterfaceStyle == UIUserInterfaceStyle.UIUserInterfaceStyleDark
+
+    //there is no an API to get a screen size in MM and calculate a real DPI
```

Comment:
What means MM ?

Reply:
mm for millimeters

Reply:
can you please write "millimeters" in this comment ?